value_test.go

```diff
@@ -935,3 +935,51 @@ func BenchmarkReadWrite(b *testing.B) {
 		}
 	}
 }
+
+// Regression test for https://github.com/dgraph-io/badger/issues/817
+func TestValueLogTruncate(t *testing.T) {
+	dir, err := ioutil.TempDir(".", "badger-test")
+	y.Check(err)
```

Comment:
require.NoError would be nice here.

Reply:
Also check if it is possible to wrap this in `runBadgerTest`, which takes care of creating directory and initializing DB.

Reply:
Using `runBadgerTest` wouldn't be useful since we want to manually open and close DB twice.